src/server_manager/install_scripts/install_server.sh

```diff
@@ -378,14 +378,18 @@ install_shadowbox() {
   readonly SB_IMAGE=${SB_IMAGE:-quay.io/outline/shadowbox:stable}
 
   log_for_sentry "Setting PUBLIC_HOSTNAME"
-  # TODO(fortuna): Make sure this is IPv4
   PUBLIC_HOSTNAME=${FLAGS_HOSTNAME:-${SB_PUBLIC_IP:-$(curl -4s https://ipinfo.io/ip)}}
```

Comment:
Let's pass `--fail` to curl. That will ensure the command exists with code 22 on http status >=400 (minus 401 and 407. See https://curl.se/libcurl/c/CURLOPT_FAILONERROR.html)

You should check the error explicitly. This will address the original issue.
We should probably assign to a PUBLIC_IP variable first, before setting the PUBLIC_HOSTNAME

Reply:
This code is now passing `--fail` and `--show-error`, so in the event of an HTTP 4XX error, the error will be printed to the console and the script will terminate immediately.